cmd/storage-rest-server.go

```diff
@@ -578,13 +576,13 @@ func (s *storageRESTServer) WalkHandler(w http.ResponseWriter, r *http.Request)
 
 	fch, err := s.storage.Walk(volume, dirPath, markerPath, recursive, r.Context().Done())
 	if err != nil {
+		logger.LogIf(r.Context(), err)
```

Comment:
don't need to log this, this is spurious @klauspost  - this will get logged if ListObjects() was triggered if bucket doesn't exist.